### ISSUE TITLE
Issue ID: DAC_Frame_Without_Title5 - Feature: YouTube embed

### ISSUE THREAD

**Issue by VDuczekW** [guardian/dotcom-rendering#4493]
Report link: https://drive.google.com/file/d/1ZXsQqmImEIx0cirPPRsaX1LKRCeoPZp5/view

Full issue list: https://docs.google.com/spreadsheets/d/1vWQtZfD49uq7uNCNELuYN_HcIj-BKiE4t4Hwda3tJaQ/edit#gid=0

**Comment by oliverlloyd** [comment]
The lack of title here is caused by Frontend [not sending the `title` value](https://github.com/guardian/frontend/blob/main/common/app/model/dotcomrendering/pageElements/PageElement.scala#L575) in the DCR model. [DCR expects it](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/lib/content.d.ts#L465) so if we were to send it over then the property would be set.

**Comment by mxdvl** [comment]
@oliverlloyd – This should be fixed with https://github.com/guardian/frontend/issues/25395
